itou/templates/layout/_header_nav_primary_items.html

```diff
@@ -30,6 +31,18 @@
                        data-matomo-action="clic"
                        data-matomo-option="header_comprendre-liae">Qui peut bénéficier des contrats d'IAE&nbsp;?</a>
                 </li>
+                {% if user.is_authenticated %}
+                    {% if user.is_employer or user.is_prescriber_with_authorized_org %}
```

Comment:
Ça m'attriste un peu de ne pas faire un `request.current_organization.is_authorized` sans requête SQL supplémentaire (mais le lien s'afficherait en fonction de l'orga sélectionnée).

Reply:
J'aurais aimer mais ça veux dire ne plus tester explicitement la partie `is_prescriber` donc ça m’embête un peu.
Déjà rien que le fait de faire 2 bloc `if` car on peux pas mixer simplement `and` et `or` me gonfle _profondément_ :sweat_smile:.